CMakeLists.txt

```diff
@@ -12,7 +12,11 @@ option(USE_VCPKG "Use VCPKG to download and manage dependencies" OFF)
 
 if (USE_VCPKG)
     # Detect correct triplet based on CMake env
-    if (WIN32)
+    if (CMAKE_SYSTEM_NAME STREQUAL "Mingw")
```

Comment:
https://cmake.org/cmake/help/latest/variable/MINGW.html there is a variable for this

Reply:
I tried using `MINGW` but it is wasn't defined.  Not sure if this is related to cross-compiling.